lib/util.js

```diff
@@ -6,11 +6,14 @@ var appPort = parseInt(argv.mocking_port, 10);
 // +1 is used for selennium
 var mocksPort = appPort + 2;
 
+var mocksHttpsPort = appPort + 3;
```

Comment:
Hi @jhudson8, if supporting both http & https is not possible, then let's just support https.  We should be moving everything to https.  Thanks!